### PR TITLE
pythonPackages.rx: init at 1.6.0

### PR DESCRIPTION
**Pull request by thanegill**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/rxpy/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, buildPythonPackage, nose }:
+
+buildPythonPackage rec {
+  pname = "rxpy";
+  name = "${pname}-${version}";
```
Comment (FRidh) [inline pkgs/development/python-modules/rxpy/default.nix]: no name

File: pkgs/development/python-modules/rxpy/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
```
Comment (FRidh) [inline pkgs/development/python-modules/rxpy/default.nix]: can't we use pypi?

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/rxpy/default.nix (hunk 2 above)
Comment (thanegill) [reply pkgs/development/python-modules/rxpy/default.nix]: Yes, Though for future is there some preferred order of sources?

Comment (thanegill) [reply pkgs/development/python-modules/rxpy/default.nix]: Just changed it to pypi. There are no tests with that source.

**Comment by thanegill** [comment]
Changed the name to match pypi (rxpy -> rx)

File: pkgs/development/python-modules/rx/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, buildPythonPackage, nose }:
+
+buildPythonPackage rec {
+  pname = "rx";
+  name = "${pname}-${version}";
```
Comment (FRidh) [inline pkgs/development/python-modules/rx/default.nix]: no `name`

Comment (thanegill) [reply pkgs/development/python-modules/rx/default.nix]: Removed name.

File: pkgs/development/python-modules/rxpy/default.nix (hunk 1 above)
Comment (thanegill) [reply pkgs/development/python-modules/rxpy/default.nix]: name has been removed

**Comment by thanegill** [comment]
ping @FRidh

**Comment by FRidh** [comment]
@thanegill this needs rebasing. Ping me again when done.

**Comment by thanegill** [comment]
@FRidh Rebased.

**Pull request merged**